profiling-crashtracking-receiver/src/main.rs

```diff
@@ -25,6 +26,8 @@ pub fn main() -> anyhow::Result<()> {
     std::io::stdin().lock().read_line(&mut metadata)?;
     let metadata: Metadata = serde_json::from_str(&metadata)?;
 
+    let telemetry_uploader = telemetry::TelemetryCrashUploader::new(&metadata, &config).ok();
```

Comment:
Do we want to do something if there is an error? Or just elide the upload?

Reply:
I'd say elide the upload. Crashtracking is best effort anyway

Reply:
We could have a metric for failed uploads. 🐢 all the way down!

Reply:
I'm a bit afraid that if the crashtracking fails, metrics wouldn't work although I guess sending a single statsd point wouldn't hurt